robot/impl/local_robot.go

```diff
@@ -71,6 +71,7 @@ type localRobot struct {
 	localPackages           packages.ManagerSyncer
 	cloudConnSvc            icloud.ConnectionService
 	logger                  logging.Logger
```

Comment:
Are we creating a convention where `logger` always refers to a diagnostics logger? The high level PR description makes sense, but I think keeping track of which loggers are which here is going to be rough.